content/src/main/content/jcr_root/apps/cru-content-importer/configurations/global/.content.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0"
+	jcr:primaryType="nt:unstructured"
+	sanitizationMap="[
+		&amp;ntilde;=&amp;amp;ntilde;,
+		&amp;ccedil;=&amp;amp;ccedil;,
+		&amp;Ccedil;=&amp;amp;Ccedil;,
+		&amp;icirc;=&amp;amp;icirc;,
+		&amp;uuml;=&amp;amp;uuml;,
+		&amp;iquest;=&amp;amp;iquest;,
+		&amp;amp;amp;#8364;=&amp;amp;euro;,
+		&amp;dDocName=&amp;amp;dDocName,
```

Comment:
I agree, this is better than the java class.

I wonder, though, is there a way you can put the list in the content of an xml element, instead of in an xml attribute? if you could, then you could user a `<![CDATA[ ... ]]>` block instead of escaping each ampersand.

Not a big deal, but maybe something to consider if you end up working with this code more in the future.